pkg/testutil/testutil.go

```diff
@@ -684,3 +684,9 @@ func Identifier(t testing.TB) string {
 	}
 	return s
 }
+
+// ImageRepo is a test helper that simply splits a full image name (i.e., not id/digest)
+// by `:` and returns the 1st part.
+func ImageRepo(s string) string {
+	return strings.Split(s, ":")[0]
```

Comment:
Doesn't seem to work when the image name contains a registry port number, e.g., `example.com:8080/example`

Reply:
I think we can use imgutil.ParseRepoTag and return the first part.